Aflores plymouth (#234)
osisoft/Edge-Data-Store-Docs: content/configuration/configuration.md

Configure each functional area of Edge Data Store to define connectivity and system behavior appropriate to your installation. The topics contained in this section provide instructions for each area.

> The examples in the Configuration topics use curl, a commonly available tool on both Windows and Linux. Any programming language or tool that supports making REST calls can be used to perform the same operations. The EdgeCmd utility can also be used to configure Edge Data Store components. To validate successful configurations, use a browser to retrieve data using GET commands.

Reviewer: ```suggestion
The examples in the Configuration topics use `curl`, a commonly available tool on both Windows and Linux. You can use any programming language or tool that supports making REST calls to perform the same operations. You can also use the EdgeCmd utility to configure Edge Data Store components. To validate successful configurations, use a browser to retrieve data using `GET` commands.
```

> The examples in the Configuration topics use `curl`, a commonly available tool on both Windows and Linux. You can use any programming language or tool that supports making REST calls to perform the same operations. You can also use the EdgeCmd utility to configure Edge Data Store components. To validate successful configurations, use a browser to retrieve data using `GET` commands.

Author: ```suggestion
The examples in the Configuration topics use curl, a commonly available tool on both Windows and Linux. You can use any programming language or tool that supports making REST calls to perform the same operations. You can also use the EdgeCmd utility to configure Edge Data Store components. To validate successful configurations, use a browser to retrieve data using `GET` commands.
```
